Makefile

```diff
@@ -68,20 +68,35 @@ SPECREVISION = $(shell echo `git describe --tags --dirty`)
 SPECREMARK = from git branch: $(shell echo `git symbolic-ref --short HEAD`) \
 	     commit: $(shell echo `git log -1 --format="%H"`)
 endif
-
-ATTRIBOPTS_NO_VERSION	= -a revdate="$(SPECDATE)" \
-			  -a revremark="$(SPECREMARK)" \
+# C++ for OpenCL doc revision scheme in aligned with its release date.
+# Revision naming scheme is as follows:
+# DOCREVYYYY.MM,
```

Comment:
Any reason for DOCREV to be in all capitals?  It feels a bit shouty.  "docrev" or "DocRev" would seem a bit friendlier to me.

Reply:
I would like to start from the upper case because it gets capitalized in HTML by default for some reasons but not in PDF.

DocRev would also be ok though...

 ```suggestion
# DocRevYYYY.MM,
```